### ISSUE TITLE
Follow logs in realtime, rather than read in chunks

### ISSUE THREAD

**Issue by seanhamlin** [lensapp/lens#2248]
**What would you like to be added**:

When you view the logs of a running container, the logs do not stream in realtime. It would be nice if this was the default (rather than polling for new logs).

**Why is this needed**:

By not following the logs in realtime, the logs seem to appear in chunks in Lens. With fast moving builds, you end up missing out on important information, as the updates seem to appear in chunks. This makes following a build extremely difficult, and you often have to back-scroll.

This is possible with a native k8s command `logs -f`, I was hoping this functionality could be implemented in Lens.

**Environment you are Lens application on:**

- Kubernetes distribution: EKS, AKS
- Desktop OS: Mac OSX 10.15

**Comment by jan-br** [comment]
Hi,
is there any ETA for this? I really love lens, but it is so hard to follow pod logs with this polling mechanism.
